octopusdeploy/resource_usernamepassword_account.go

```diff
@@ -0,0 +1,126 @@
+package octopusdeploy
+
+import (
+	"fmt"
+
+	"github.com/OctopusDeploy/go-octopusdeploy/octopusdeploy"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceUsernamePassword() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceUsernamePasswordCreate,
+		Read:   resourceUsernamePasswordRead,
+		Update: resourceUsernamePasswordUpdate,
+		Delete: resourceUsernamePasswordDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"environments": {
+				Type: schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Optional: true,
+			},
```

Comment:
The tenant scoping is missing from this resource. I think the common elements of Account should be moved into re-used method like we did with the [actions](https://github.com/OctopusDeploy/terraform-provider-octopusdeploy/blob/0ecd0b5184f2b850c9a507c7d340e1b7ec6fbb17/octopusdeploy/manual_intervention_action.go#L9).

Reply:
Sounds good. I added back in the tenant so it'll use `getTenantedDeploymentSchema()`